src/interface.jl

```diff
@@ -25,16 +25,26 @@ function AFM(R::Real)
 end
 
 """
-    Tweezers(R::Real)
+    Tweezers(R::Real, slip::Real)
 
-Create and return an `Interface` struct for a spherical probe of radius `R` embeded in a material.
+Create and return an `Interface` struct for a spherical probe of radius `R` embeded in a material, with probe/material viscosity ratio `slip`.
 """
-function Tweezers(R::Real)
-  Interface(:d, :f, (ϵ,σ)->(d = R .* ϵ, f = R^2 .* σ ), (d,f)->(ϵ = d ./ R, σ = f ./ R^2) )
-end
+function Tweezers(R::Real, slip::Real)
+  # slip can go between 0 and Inf 
+  # no slip is at Inf, frictionless slip is at 0
+  R >= 0 || error("sphere must have a positive radius")
+  slip >= 0 || error("slip less than 0 is not physical")
 
+  if slip == Inf
+    hadamard = 3. / 2.
+  else
+    hadamard = (((3 .* slip) + 2)./ (2 .* (slip + 1)))
```

Comment:
when slip is 0, we get hadamard = 1. When slip is 1, 5/4. Shouldn't this be 3/2 ? 
`hadamard = (((4 * slip) + 2) / (2 * (slip + 1)))`  ?

The broadcast operations are not needed here as there is no array to process.

Reply:
OK, that makes sense now. 
Slip in the range 0-Inf. We'll set the default value to Inf. Should not be a problem in python/R as one doesn't need to enter any value for the no slip condition.